scripts/verify-release.sh

```
@@ -0,0 +1,118 @@
+      echo "-> Downloading release ${tag} to ${workdir}"
+      (
+        cd "${workdir}"
+        gh release download "${tag}" \
+          --repo "${REPO}" \
+          --pattern '*.tgz' \
+          --pattern '*.sig' \
+          --pattern '*.cert' \
+          --pattern 'provenance.intoto.jsonl' \
+          || { echo "error: release assets missing — exit 5" >&2; exit 5; }
+      )
+      tarball="$(ls "${workdir}"/*.tgz 2>/dev/null | head -1)"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)